### ISSUE TITLE
docker file error

### ISSUE THREAD

**Issue by dbabox** [BrosSquad/FastEndpoints.Template#3]
docker file error, it   continue using aspnet6

**Comment by CerealKiller97** [comment]
@dbabox True, Template has not been updated recently, I am quite busy lately, when I find free time, I'll fix it.
